adx/resource_adx_table_update_policy.go

```diff
@@ -0,0 +1,118 @@
+package adx
+
+import (
+	"context"
+	"fmt"
+
+	"encoding/json"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type TableUpdatePolicy struct {
+	IsEnabled bool
+	Source string
+	Query string
+	IsTransactional bool
+	PropagateIngestionProperties bool
+}
+
+func resourceADXTableUpdatePolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceADXTableUpdatePolicyCreate,
+		ReadContext:   resourceADXTableUpdatePolicyRead,
+		DeleteContext: resourceADXTableUpdatePolicyDelete,
+		UpdateContext: resourceADXTableUpdatePolicyCreate,
```

Comment:
At least by azurerm convention - if we use the same func to do both Create and Update - func name is `resourceXXXCreateUpdate`. Could we amend that please?

Reply:
fixed